pyms/flask/services/swagger.py

```diff
@@ -1,8 +1,12 @@
 import os
+from pathlib import Path
+from typing import Dict, Any
 
 import connexion
 from connexion.resolver import RestyResolver
 
+import prance
```

Comment:
prance mus be loaded like https://github.com/python-microservices/pyms/blob/master/pyms/flask/services/tracer.py to prevent import errors if we don't use swagger in our project and not install py-ms with `py-ms[all]`